.github/index.pkl

```diff
@@ -119,8 +119,7 @@ prb {
             "native-\(job.project)-\(job.os)",
             "native-\(job.project)-\(job.os)-\(job.arch)",
           )
-        `if` =
-          tags.map((it) -> "startsWith(github.event.comment.body, '/build \(it)')").join(" || ")
+        `if` = tags.map((it) -> "contains(github.event.pull_request.body, '/build \(it)')").join(" || ")
```

Comment:
[`Context.github.event("pull_request.body")`](https://github.com/StefMa/pkl-gha/blob/7df223000cacc1ca06c4a412a5808402298afc0a/Context.pkl#L60) 🤓 

```suggestion
        `if` = tags.map((it) -> "contains(\(Context.github.event("pull_request.body")), '/build \(it)')").join(" || ")
```

Reply:
I don't think this is right since the `Context` APIs produce template tags (eg. `${{ github.even.pull_request.body }}`) but `if` should omit the tags.